client/client-benchmark-noop-api-plugin/src/main/java/org/elasticsearch/plugin/noop/NoopPlugin.java

```diff
@@ -43,7 +51,11 @@
     }
 
     @Override
-    public List<Class<? extends RestHandler>> getRestHandlers() {
-        return Arrays.asList(RestNoopBulkAction.class, RestNoopSearchAction.class);
+    public List<RestHandler> initRestHandlers(Settings settings, RestController restController, ClusterSettings clusterSettings,
```

Comment:
See commit message for my I did this instead of some fancy ctor reference stuff.